test/include.mk

```diff
@@ -104,13 +104,13 @@ PATTERN_OPTS = --pattern "$$(cat $*.k)"
 	@echo "KORE_EXEC_OPTS =" $(KORE_EXEC_OPTS)
 	rm -f $@
 	$(if $(STORE_PROOFS),rm -f $(STORE_PROOFS),$(if $(RECALL_PROOFS),cp $(RECALL_PROOFS) $(@:.out=.save-proofs.kore)))
-	$(KOMPILE) $(KOMPILE_OPTS) --main-module $(KPROVE_MODULE) $(KPROVE_SPEC)
-	rm -rf $*-tmpdir; mkdir $*-tmpdir
-	mv $*-spec-kompiled $*-tmpdir
-	$(KPROVE) $(KPROVE_OPTS) -d $*-tmpdir $(KPROVE_SPEC_OPTS) $(KPROVE_SPEC) >$@ || true
+#	$(KOMPILE) $(KOMPILE_OPTS) --main-module $(KPROVE_MODULE) $(KPROVE_SPEC)
+#	rm -rf $*-tmpdir; mkdir $*-tmpdir
+#	mv $*-spec-kompiled $*-tmpdir
```

Comment:
Did you forget to remove the commented out lines here?

Reply:
Yes